client/scripts/legic.lua

```diff
@@ -2179,6 +2156,7 @@ function checkKghCrc(tag, segid)
   if (type(tag.SEG[segid])=='table') then
     if (tag.data[3]=="11" and tag.raw=="9f" and tag.SSC=="ff") then
       local data=kghCrcCredentials(tag, segid)
+      print("CRC:"..data.."  --  "..("%02x"):format(utils.Crc8Legic(data)).." == "..tag.SEG[segid].data[tag.SEG[segid].len-5-1])
       if (("%02x"):format(utils.Crc8Legic(data))==tag.SEG[segid].data[tag.SEG[segid].len-5-1]) then return true; end
```

Comment:
The question if the tag-bytedata is as a string,  and compares with UPPER/LOWER values.. 
or is the crc a value comparision here..   I'm not sure.